ensemble/volren/volume_data_filter.py

```diff
@@ -0,0 +1,11 @@
+class VolumeFilter():
```

Comment:
The name of the file should be the same as the name of the class. Also, this class is old-style: at a minimum, it should inherit from `object`, but we will want to inherit from `HasStrictTraits` since subclasses will be much more complex.